### PR TITLE
fininshed code challenge

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/main/java/code401/challenges/tree/BinaryTree.java [hunk 1]
```diff
@@ -86,7 +103,38 @@ public void postOrderHelper(Node<T> node, ArrayList<T> resultArr){
...
+    void printGivenIndex (Node root ,int currentLevel, StringBuilder resultStr)
+    {
+        if (root == null)
+            return;
+        if (currentLevel == 1){
+            resultStr.append(root.value);
+            resultStr.append(" ");
+        }
+        else if (currentLevel > 1)
+        {
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/tree/BinaryTree.java:132]: This should be at the end of the previous line!
`else if (currentLevel > 1) {`
